TictactoeVer2Tests/TictactoeTests.cs

```diff
@@ -88,32 +118,36 @@ public void ShouldStartGameWithPlayer1AndPlayer2()
         [Fact]
         public void ShouldStartGameWithZeroInitialScores() // TODO change test to behavioural test
         {
+            InitializeTictactoeGame();
             StartGameWith3X3Board();
             
             Assert.Equal(0, Game.Player1.Score);
             Assert.Equal(0, Game.Player2.Score);
         }
         
         [Fact]
-        public void ShouldNotStartWithNoPlayerWhenGameNotStarted()
+        public void ShouldNotStartWithPlayerXWhenGameNotStarted()
```

Comment:
Unnecessary "negative" test given that we have tests to check what should happen in this case